describe/EnumVal.ts

```diff
@@ -0,0 +1,20 @@
+export type EnumValDescription = {
+  /** the attribute `ord` of the element `EnumVal` */
+  ord?: number;
+  /** the attribute `desc` of the element  `EnumVal` */
+  desc?: string;
+  /** the textContent of the element `EnumVal` */
+  content?: string;
```

Comment:
"content" sounds a bit vague to me. Do you think `val` might be a clearer name?

Reply:
Thank you yes!